tests/unittests/gradCheckTest.cpp

```diff
@@ -196,22 +201,22 @@ static void gradientCheckGroupConv(size_t numInputChan, size_t group) {
   inputsH.initXavier(1, mod.getPRNG());
   outputsH.initXavier(1, mod.getPRNG());
 
-  performGradCheck(IP, result, A, Ex, &inputs, &outputs, 0.001, 0.004);
+  performGradCheck(EE_, result, A, Ex, &inputs, &outputs, 0.001, 0.004);
 }
 
-TEST(Network, gradientCheckConv) { gradientCheckGroupConv(1, 1); }
+TEST_P(GradCheck, gradientCheckConv) { gradientCheckGroupConv(1, 1, EE_); }
```

Comment:
are not all those groupConv tests? (names of the tests seems odd, is it really gradientCheckDepthwiseConv)

Reply:
The test are updated by https://github.com/pytorch/glow/commit/361dc5a156496f1d25083cec70f82335cdf6c673#diff-1c1a40d1f65512890dcbc5053e5dd5a3. 
I think the first one is just keep the original test (non-group). The second one is called DepthwiseConv, for depethwise conv, all I know so far is that the number of group should be equal to the input channel.  I need to do more research on this part.